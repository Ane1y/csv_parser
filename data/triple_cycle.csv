,       Cell,    V,     rot
1,      10,     =Cell70, 8
70,    =V4,     9,       =Cell1
4,      3 ,     =V1,     8