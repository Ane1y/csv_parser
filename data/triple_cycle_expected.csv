,Cell,V,rot
1,10,[ERR],8
70,[ERR],9,10
4,3,[ERR],8
